coolify/haproxy/Dockerfile

```
@@ -0,0 +1,12 @@
+FROM haproxy:2.9-alpine
+
+# Switch to root to install packages
+USER root
+
+# Install curl for healthcheck
+RUN apk add --no-cache curl
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)